src/log/ModUdp.cc

```diff
@@ -44,6 +44,7 @@ logfile_mod_udp_write(Logfile * lf, const char *buf, size_t len)
     s = write(ll->fd, (char const *) buf, len);
     fd_bytes(ll->fd, s, FD_WRITE);
 #if 0
+    // TODO: reduce verbosity to 3 and use a FadingCounter to reduce spamminess
```

Comment:
That is not a good suggestion because FadingCounter should not be used for level-3 messages. We can discuss an appropriate design, which might even include FadingCounter usage, but let's not do it here and now. We can just leave a more general comment for now:

```suggestion
    // TODO: Enable after polishing to properly log these errors.
```

I do _not_ insist on this change or any specific replacement text, but if somebody follows the current TODO instructions, they will have to rewrite their code and will probably be unhappy about that.

Reply:
Thanks for the suggestion